src/style.css

```diff
@@ -1,3 +1,67 @@
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
 body {
+  margin: 0 auto;
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+.header {
+  padding: 0 50px ;
+  display: flex;
+  position: relative;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+.logo {
+  display: flex;
+  margin: 10px 0 ;
+}
+
+.link {
+  display: inline-block;
+  margin-right: 20px;
+
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
   margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  padding: 23px 0;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  height: 60px;
+  line-height: 60px;
+  position: relative;
+}
+
+.is-active,
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)